atom/atom
# Migrate 'atom-ui' into core

> ⚠️ Wait with merging until the next release is out, so we can test this a bit longer.

### Description of the Change

This PR moves [atom-ui](https://github.com/atom/atom-ui) back into `static/`. In addition, core styles and icons got their own folders.

- [x] Move `atom-ui` styles
- [x] Remove `atom-ui` dependency
- [x] Move core styles to `static/core-ui/`
- [x] Move icons to `static/icons/`

After this PR is merged.

- [x] Archive the [atom/atom-ui](https://github.com/atom/atom-ui) repo.

### Benefits

It's part of the package migration initiative.

Originally `atom-ui` got moved to its own repo in hopes of increased community contribution. Unfortunately that didn't happen. Also, once the `styleguide` package gets migrated, it will be easier to make updates since they are both part of the same repo.

### Possible Drawbacks

There is a chance for regressions.

### Verification Process

1. Open Atom in dev mode: `atom . -d`
2. Verify that the UI still looks the same.
3. Verify that there are no errors in the console.
4. Verify that the icons get rendered.

### Applicable Issues

N/A

### Release Notes

N/A
